### ISSUE TITLE
clock

### ISSUE THREAD

**Issue by jgallen23** [progrium/envy#43]
I've noticed my clock regularly gets out of sync in envy environments.  Any good way to fix this?  Is there a way for docker to use the system's clock?

**Comment by progrium** [comment]
Are you running in boot2docker?
https://github.com/boot2docker/boot2docker/issues/290

**Comment by jgallen23** [comment]
no, running on ubuntu 14.04

**Comment by progrium** [comment]
And your host is not also out of sync? 
http://stackoverflow.com/questions/24551592/how-to-make-sure-dockers-time-syncs-with-that-of-the-host

**Comment by jgallen23** [comment]
nope, host not out of sync.  I'll keep an eye out for when it happens again
